components/appmgt/org.wso2.carbon.appmgt.gateway/src/main/java/org/wso2/carbon/appmgt/gateway/handlers/proxy/ReverseProxyHandler.java

```diff
@@ -1,6 +1,22 @@
-package org.wso2.carbon.appmgt.gateway.handlers.proxy;
+/*
+*  Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
```

Comment:
Sure will do it. This PR is quite old (Apr 24, 2015) and at that time we used year range for the license.